packages/common/src/sites/feeds/_internal/defaults.ts

```diff
@@ -31,7 +31,7 @@ const DCAT_US_1X_DEFAULT = {
     fn: "{{owner}}",
     hasEmail: "{{orgContactEmail}}",
   },
-  spatial: "{{extent}}",
+  spatial: "{{extent:computeSpatialProperty}}",
```

Comment:
This will work for _new_ sites, or sites that don't have a dcat configuration previously saved. What do we want to do about sites that have "spatial: "{{extent}}" hard coded in their saved templates?

Reply:
> What do we want to do about sites that have "spatial: "{{extent}}" hard coded in their saved templates?

We will have to request those customers to update their saved templates manually through feed editor. The sites affected by the bug are the ones that don't have dcat configurations saved.